lib/search.js

```diff
@@ -148,15 +168,22 @@ module.exports = new class LibSearch {
                 score: 0
             }
 
+            // imdb, max score 7
+            if (input.imdbid && sub.imdbid === input.imdbid)
+                tmp.score += 5
+            if (input.season && input.episode && sub.season === input.season && sub.episode === input.episode)

```

Comment:
using double operators `==` over triple `===` might be better to avoid string/integer errors. `1 == '1' => true` but `1 === '1' => false`